CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Current release:
   * [CHANGELOG-1.5.md][15]
 
+## Development releases:
+  * [Unreleased changes][16]
```

Comment:
suggest matching this w/ line 1 of `changelogs/CHANGELOG-1.6.md`

Reply:
Thanks, I wasn't sure what to use here but based it on a previous RC/Beta PR. I originally put in the new version but changed it :) Just to confirm, did you mean using "v1.6.0-rc.1" here instead of "Unreleased changes"?

Reply:
Yes. 🙂